src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -28,15 +28,15 @@ import (
 	"path/filepath"
 	"sort"
 	"strings"
-	"syscall"
 	"unsafe"
 
 	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/windows"
-	"golang.org/x/text/encoding/unicode"
 )
 
+const CREATE_NO_WINDOW = 0x08000000
```

Comment:
Is this used? (The only `CREATE_NO_WINDOW` in this file refers to the export from `windows`, which makes sense…)

Reply:
Whoops, I missed that. Fixed!